firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -1097,6 +1106,12 @@ public void testSpecTests() throws Exception {
       parsedSpecFiles.add(new Pair<>(f.getName(), fileJSON));
     }
 
+    String exclusiveTestNameFromSystemProperty =
+        emptyToNull(System.getProperty(EXCLUSIVE_PROPERTY));
```

Comment:
If you go with regular expression approach, this could be `System.getProperty(EXCLUSIVE_PROPERTY, ".*")`, in which case you can always apply the filter.

Reply:
This won't work though because I need to be able to distinguish between the property being set or not being set.  The reason is that if it is set then `exclusiveMode` must be set to `true` and if it is not then it must be left alone.  By using a default value I cannot know if .* was specified explicitly or was the default value.